fixtures/README.md

```diff
@@ -0,0 +1,36 @@
+# Fixtures
+
+This folder contains sample projects we use in the integration and acceptance tests. 
+Please keep this keep in alphabetical order.
+
+## app_with_frameworks
+
+Slightly more complicated project consists of an iOS app and few frameworks. 
+
+#### Structure
+
+```
+Workspace:
+  - App:
+    - MainApp (iOS app)
+    - MainAppTests (iOS unit tests)
+  - Framework1:
+    - Framework1 (dynamic iOS framework)
+    - Framework1Tests (iOS unit tests)
+  - Framework2:
+    - Framework2 (dynamic iOS framework)
+    - Framework2Tests (iOS unit tests)
+```
+
+Dependencies:
+  - App -> Framework1
+  - App -> Framework2
+  - Framework1 -> Framework2
+
+## app_with_tests
```

Comment:
What do you think about including the app platform in the name? `ios_app_with_tests`?

Reply:
Sure, but I guess sooner of later we will have some projects with macOS and iOS targets, or even tvOS and watchOS and then it might be a bit tricky to keep the naming consistent. Maybe we can try to include all the platforms in the alphabetical order?

Reply:
> Maybe we can try to include all the platforms in the alphabetical order?

That sounds good 👍